corehq/apps/api/urls.py

```diff
@@ -172,15 +172,17 @@ def api_url_patterns():
 USER_API_LIST = GLOBAL_USER_API_LIST + NON_GLOBAL_USER_API_LIST
 
 
-def get_global_api_url_patterns(resources):
+def _get_global_api_url_patterns(resources):
     api = CommCareHqApi(api_name='global')
     for resource in resources:
         api.register(resource())
-        yield url(r'^', include(api.urls))
```

Comment:
I bet this was originally an overlooked indentation error, where this line was supposed to be one level less indented. Nice catch!

Reply:
In that case `return` would make more sense than `yield` since the function would only ever yield a single item. It's been this way since 2014.